httpserver/http_connection.cpp

```diff
@@ -125,10 +123,20 @@ void LokidClient::make_custom_lokid_request(const std::string& daemon_ip,
     make_http_request(ioc_, daemon_ip, daemon_port, req, std::move(cb));
 }
 
-private_key_t LokidClient::wait_for_privkey() {
+static bool validateHexKey(const std::string& key) {
+    return key.size() == 2 * loki::KEY_LENGTH &&
+           std::all_of(key.begin(), key.end(), [](char c) {
+               return (c >= '0' && c <= '9') || (c >= 'a' && c <= 'f');
```

Comment:
Uppercase `(c >= 'A' && c <= 'F')`?

Reply:
The underlying functions expect lower case, so I will leave it as is for now.